db/stats.go

```diff
@@ -78,6 +79,47 @@ func GetNftCreatorCount(conn *pgxpool.Conn) (count QueryCountResponse, err error
 	return
 }
 
+func GetNftRecentCreatorCount(conn *pgxpool.Conn, q QueryNftRecentCreatorCountRequest) (res QueryNftRecentCreatorCountResponse, err error) {
+	ts := time.Now().Unix()
+	if q.Time != nil {
+		ts = int64(*q.Time)
+	}
+	ts -= int64(q.PeriodDays * 24 * 60 * 60)
+
+	sql := `
+	WITH recent_creators AS (
+		SELECT DISTINCT sender FROM nft_event
+		WHERE action = 'new_class'
+			AND timestamp >= to_timestamp($1)
+	),
+	early_creators AS (
+		SELECT DISTINCT sender FROM nft_event
+		WHERE action = 'new_class'
+			AND timestamp < to_timestamp($1)
+	)
+
+	SELECT 
+		(SELECT COUNT(DISTINCT rc.sender)
+			FROM recent_creators AS rc
+			LEFT JOIN early_creators AS ec 
+				ON rc.sender = ec.sender
+			WHERE ec.sender IS NULL) AS new_creator_count,
+		(SELECT COUNT(DISTINCT rc.sender)
+			FROM recent_creators AS rc
+			JOIN early_creators AS ec 
+				ON rc.sender = ec.sender) AS returning_creator_count;
```

Comment:
Seems we only need to know about address that exists in `recent_creators`, but `early_creators` query the whole table
Would we do better with LATERAL query than self join (or sth)?

Reply:
Per Edmond's use case, I return the all-time result with another query.
See if it's OK:
https://node.testnet.like.co/statistics/nft/returning-creator-count?period_days=30

Reply:
Nice to use window function, but I dont think we should hardcode all data in weeks interval
Just use  (period_days) as window interval, return latest 1 entry by default, and accept a start/end time as param if really needed?

Reply:
Say for period_days=30, we should use return the data for "now-30days to now"